src/foam/core/AbstractPropertyInfo.java

```diff
@@ -36,6 +38,16 @@ public void toJSON(foam.lib.json.Outputter outputter, StringBuilder out, Object
   }
 
   @Override
+  public String createStatement() {
+    return getName();
+  }
+
+  @Override
+  public Object prepareStatement() {
```

Comment:
prepareStatement should return void and be passed the prepared statement. This method should do nothing.